### ISSUE TITLE
Multisig redeeming results in a wallet crash + corrupted block data/wallet.dat

### ISSUE THREAD

**Issue by LyGhT1337** [bulwark-crypto/Bulwark#143]
**Describe the bug**
You will get a crash when you try to redeem a multisig redeemScript.

**To Reproduce**
1. Go to File > Multisig Creation
2. Put in your Redeem key in the bottom field.
3. Click on Import Redeem

**Expected behavior**
As soon as you try to redeem the Script your wallet will crash with either your Block files or even your wallet.dat being corrupted and with an error.
Error below under Screenshots

**Screenshots**
https://cdn.discordapp.com/attachments/394629520843472906/523552829307093012/unknown.png

**Desktop (please complete the following information):**
- Os : Windows 10 latest build
 - Version 2.0

**Log:**

Log got renewed after the next start, means its gone D:

**Comment by KaneoHunter** [comment]
I believe this could be resolved with some UX improvements, doing some research I can see where the confusion came from.

1. There's currently no easy way to find a previously generated pay-to-multisig scripts, without knowledge of how P2MS works and it's format to allow a user to manually create it. It would be useful if the multisig dialogue could re-create the ScriptPubKey in addition to the error when creating a address already found inside the wallet. This was the root cause of our issue because I had lost the redeem key and couldn't generate it again to pass it on to @LyGhT1337 .

2. Being inquisitive, I went online to find a debug console command to try and force the wallet to generate the P2MS redeem again, and stumbled on `createmultisig`, not knowing much about the topic, this lead me to generating a P2SH redeem which I passed to Lyght. Given that we have the GUI in place, may we want to consider disabling the `createmultisig` command to prevent any user confusion in the future? Maybe not, but something to consider.

3. I don't believe that a formatting error should cause a fatal wallet crash, I feel a useful task to have in the background would be setting up data validation to catch any failed inputs to prevent wallet crashes if someone messes up the redeem on the multisig GUI.

I could be mistaken in what I'm saying, so feel free to correct any inaccuracies.
